python/cuml/test/experimental/test_explainer_kernel_shap.py

```diff
@@ -399,6 +416,11 @@ def test_l1_regularization(exact_tests_dataset, l1_type):
      0.00088981]
 ]
 
+housing_regression_result = np.array(
```

Comment:
Was this obtained by running shap? Would be good to note in a comment what you did to get it and what version you used.